src/components/admin/ImageController.jsx

```diff
@@ -138,6 +148,14 @@ export default class ImageUploader extends BaseComponent {
       this.addImagePreviewUrl(file);
     });
 
+    if (tooLargeFileNames.length !== 0) {
+      let names = '\n';
+      tooLargeFileNames.forEach((file) => {
+        names = names.concat(` ${file.name} (${(file.size / 1024).toFixed(2)} KB)\n`);
+      });
+      alert(`The following exceeded the maximum upload size: ${names}`);
```

Comment:
Maybe just make this `the following files...` (adding in "files").

Also, it would generally be a lot more natural to add the newline in here instead of as the first part of the string above, so just making it `size:\n${names}`), but with my above approach you can do it all in one and just do `alert(alertMessage)` here.

Reply:
Instead of `alert`, let's use a Material UI [dialog box](http://www.material-ui.com/#/components/dialog). This falls in line with the rest of the Admin Interface and is just a little less jarring than an `alert` message.

Reply:
Yeah you're right, let's keep things pretty! I'll try not to fall into my old habits ;)